### PR TITLE
Dave

### PR DESCRIPTION
**Pull request by dzagardo**
Took Chris's extension and made changes. Some stuff can probably be deleted.

Main files changed:

Manifest.json
popup.html
datadaddyccpa.png

### CODE REVIEW

File: hw1/chris_extension.pem [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (cdchoy) [inline hw1/chris_extension.pem]: ahh!! don't push your private key to public repos!

**Comment by cdchoy** [comment]
Update your .gitignore file to include '.DS_Store' so that that file isn't staged for commits. It's a file purely for local machine file organization on MacOS and it can mess things up for other people's machines if pulled down. Please avoid committing it to the team repo.

**Review by dzagardo — COMMENTED** [review]
Revisions made.

File: .gitignore [hunk 2]
```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# DS store
+.DS_Store
+
+# skip extension private key
+hw1/chris_extension.pem
```
Comment (cdchoy) [inline .gitignore]: can change to `*.pem` to be generic so future private keys aren't staged without needing to update the gitignore for all

**Review by cdchoy — COMMENTED** [review]
Use `git rm --cached .DS_Store` and `git rm --cached */.DS_Store` to untrack the already tracked .DS_Store files to prevent weird behavior with future commits.

https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign

**Comment by dzagardo** [comment]
already did before pushing!

Screen Shot 2022-02-22 at 3 20 54 PM

**Review by dzagardo — COMMENTED** [review]
Revision requests addressed.

**Review by cdchoy — APPROVED** [review]
Not sure why the .DS_Store files are still being merged in. Maybe bc they were already pushed? idk. Git is weird and I'm not going to keep blocking the PR to resolve this.

**Pull request merged**